sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Listeners/ServiceBusListener.cs

```diff
@@ -363,36 +357,32 @@ internal async Task ProcessSessionMessageAsync(ProcessSessionMessageEventArgs ar
 
             _concurrencyUpdateManager?.MessageProcessed();
 
-            using (CancellationTokenSource linkedCts =
-                CancellationTokenSource.CreateLinkedTokenSource(args.CancellationToken, _cancellationTokenSource.Token))
+            var actions = new ServiceBusSessionMessageActions(args);
+            if (!await _sessionMessageProcessor.Value.BeginProcessingMessageAsync(actions, args.Message, args.CancellationToken)
+                .ConfigureAwait(false))
             {
-                var actions = new ServiceBusSessionMessageActions(args);
-                if (!await _sessionMessageProcessor.Value.BeginProcessingMessageAsync(actions, args.Message, linkedCts.Token)
-                    .ConfigureAwait(false))
-                {
-                    return;
-                }
+                return;
+            }
 
-                var receiveActions = new ServiceBusReceiveActions(args);
-                ServiceBusTriggerInput input = ServiceBusTriggerInput.CreateSingle(args.Message, actions, receiveActions, _client.Value);
+            var receiveActions = new ServiceBusReceiveActions(args);
+            ServiceBusTriggerInput input = ServiceBusTriggerInput.CreateSingle(args.Message, actions, receiveActions, _client.Value);
 
-                TriggeredFunctionData data = input.GetTriggerFunctionData();
-                FunctionResult result = await _triggerExecutor.TryExecuteAsync(data, linkedCts.Token).ConfigureAwait(false);
+            TriggeredFunctionData data = input.GetTriggerFunctionData();
+            FunctionResult result = await _triggerExecutor.TryExecuteAsync(data, CancellationToken.None).ConfigureAwait(false);
```

Comment:
This should still be passing a listener-level token that is cancelled in Disposed.